docs/reference/tools/pd-control.md

```diff
@@ -98,48 +98,119 @@ Use this command to view or modify the configuration information.
 Usage:
 
 ```bash
->> config show                                // Display the config information of the scheduler
+>> ./bin/pd-ctl config show all   // Display all config information of the scheduler
 {
-  "max-snapshot-count": 3,
-  "max-pending-peer-count": 16,
-  "max-merge-region-size": 20,
-  "max-merge-region-keys": 200000,
-  "split-merge-interval": "1h0m0s",
-  "patrol-region-interval": "100ms",
-  "max-store-down-time": "1h0m0s",
-  "leader-schedule-limit": 4,
-  "region-schedule-limit": 4,
-  "replica-schedule-limit":8,
-  "merge-schedule-limit": 8,
-  "tolerant-size-ratio": 5,
-  "low-space-ratio": 0.8,
-  "high-space-ratio": 0.6,
-  "disable-raft-learner": "false",
-  "disable-remove-down-replica": "false",
-  "disable-replace-offline-replica": "false",
-  "disable-make-up-replica": "false",
-  "disable-remove-extra-replica": "false",
-  "disable-location-replacement": "false",
-  "disable-namespace-relocation": "false",
-  "schedulers-v2": [
-    {
-      "type": "balance-region",
-      "args": null,
-      "disable": false
+  "client-urls": "http://127.0.0.1:2379",
+  "peer-urls": "http://127.0.0.1:2380",
+  "advertise-client-urls": "http://127.0.0.1:2379",
+  "advertise-peer-urls": "http://127.0.0.1:2380",
+  "name": "pd-chenshunings-MacBook-Pro.local",
+  "data-dir": "default.pd-chenshunings-MacBook-Pro.local",
+  "force-new-cluster": false,
+  "initial-cluster": "pd-chenshunings-MacBook-Pro.local=http://127.0.0.1:2380",
+  "initial-cluster-state": "new",
+  "join": "",
+  "lease": 3,
+  "log": {
+    "level": "",
+    "format": "text",
+    "disable-timestamp": false,
+    "file": {
+      "filename": "",
+      "log-rotate": true,
+      "max-size": 0,
+      "max-days": 0,
+      "max-backups": 0
     },
-    {
-      "type": "balance-leader",
-      "args": null,
-      "disable": false
-    },
-    {
-      "type": "hot-region",
-      "args": null,
-      "disable": false
-    }
-  ]
+    "development": false,
+    "disable-caller": false,
+    "disable-stacktrace": false,
+    "sampling": null
+  },
+  "log-file": "",
+  "log-level": "",
+  "tso-save-interval": "3s",
+  "metric": {
+    "job": "pd-chenshunings-MacBook-Pro.local",
+    "address": "",
+    "interval": "15s"
+  },
+  "schedule": {
+    "max-snapshot-count": 3,
+    "max-pending-peer-count": 16,
+    "max-merge-region-size": 20,
+    "max-merge-region-keys": 200000,
+    "split-merge-interval": "1h0m0s",
+    "enable-two-way-merge": false,
```

Comment:
```suggestion
    "enable-one-way-merge": false,
```

Reply:
Should these two be included in the PD-ctl command description?